pallets/configuration/src/lib.rs

```diff
@@ -131,7 +131,7 @@ impl HostConfiguration {
         if self.max_orchestrator_collators < self.min_orchestrator_collators {
             return Err(InconsistentError::MaxCollatorsLowerThanMinCollators);
         }
-        if self.collators_per_parathread != 1 || self.parathreads_per_collator != 1 {
+        if self.parathreads_per_collator != 1 {
```

Comment:
here we are finally assuming that we always have one collator per parathread right?

Reply:
No, here we allow more than one collator per parathread (just like with parachains), but we don't allow one collator to be assigned to more than one parathread (at least not yet).